vms/platformvm/service.go

```diff
@@ -110,34 +113,43 @@ type ImportKeyArgs struct {
 func (service *Service) ImportKey(r *http.Request, args *ImportKeyArgs, reply *api.JsonAddress) error {
 	service.vm.SnowmanVM.Ctx.Log.Info("Platform: ImportKey called for user '%s'", args.Username)
 
+	db, err := service.vm.SnowmanVM.Ctx.Keystore.GetDatabase(args.Username, args.Password)
+	if err != nil {
+		return fmt.Errorf("problem retrieving data: %w", err)
+	}
+
+	user := user{db: db}
+	if addrs, _ := user.getAddresses(); len(addrs) >= maxKeystoreAddresses {
+		_ = db.Close()
```

Comment:
Rather than explicitly calling `_ = db.Close()` I think we should just `defer` the closing of the db after it is created.

Reply:
Sounds good, just pushed an update